charts/avatar-creation/values.yaml

```diff
@@ -57,6 +57,7 @@ env:
   MEASURE_HOST: measure-engine
   BUCKET: avatar-creation-dev
   REGION: us-west-2
+  AVATAR_CONTROLLER_HOST: avatar-api
```

Comment:
Shouldn't the env var name be `AVATAR_API_HOST` ?

Reply:
i would have to make that change in code, its fine the changed value should work

Reply:
I can add this change later, the repo name is also avatar-controller too, I would have to change that too

Reply:
You should change it everywhere. We can't allow these kinds of these to get in. This is technical debt. Naming is very important as to not confuse people.